### ISSUE TITLE
Literal space causes false positive from UnneededCapitalW #1263

### ISSUE THREAD

**Issue by chuckd** [bbatsov/rubocop#3334]
#1263 fixed Style/UnneededCapitalW to allow word arrays that include special characters and escape strings, however literal space (\s) still causes a false positive.

```ruby
%W(\s)
```

It's a slightly strange use of a word array I guess but in my case the easiest given a long array where one or two elements need to be a blank string.

I had a go at fixing myself, it's something to do with the double_quotes_acceptable? method, but couldn't see how obviously to change it.

```
$ rubocop -V
0.40.0 (using Parser 2.3.1.2, running on ruby 2.3.1 x86_64-darwin15)
```
